src/core/thread/mle.cpp

```diff
@@ -670,6 +665,22 @@ otError Mle::SetStateChild(uint16_t aRloc16)
     return OT_ERROR_NONE;
 }
 
+void Mle::InformPreviousChannel(void)
+{
+    VerifyOrExit(mPreviousPanId != Mac::kPanIdBroadcast);
```

Comment:
Should we validate that the device is attached?

Reply:
Updated. Thanks.